# Default initial positions for NEXTAGE's ros2_control fake system

initial_positions:
  RARM_JOINT0: 0.0
  RARM_JOINT1: 0.0
  RARM_JOINT2: 0.0
  RARM_JOINT3: 0.0
  RARM_JOINT4: 0.0
  RARM_JOINT5: 0.0
  LARM_JOINT0: 0.0
  LARM_JOINT1: 0.0
  LARM_JOINT2: 0.0
  LARM_JOINT3: 0.0
  LARM_JOINT4: 0.0
  LARM_JOINT5: 0.0